cfgov/jobmanager/jinja2/jobmanager/job_listing_details.html

```diff
@@ -0,0 +1,133 @@
+{# ==========================================================================
+
+   Job Listing Details
+
+   ==========================================================================
+
+   Description:
+
+   Renders job details when given:
+
+   value.title:             Job title.
+
+   value.division:          Job division.
+
+   value.close_date:        Date job closes.
+
+   value.offices:           List of offices, where each office has:
+
+   office.name:             Office city name.
+
+   office.state_id:         Office state abbreviation.
+
+   value.remote_allowed:    Whether an office job supports working remotely.
+
+   value.regions:           List of regions, where each region has:
+
+   region.name:             Region name.
+
+   region.states:           List of states in region, as strings.
+
+   region.major_cities:     List of major cities in region, where each city has:
+
+   major_city.name:         Major city name.
+
+   major_city.state_id:     Major city state abbreviation.
+
+   value.grades:            List of job grades, as integers.
+
+   value.salary_min:        Minimum salary, as float.
+
+   value.salary_max:        Maximum salary, as float.
+
+   value.applicant_types:   List of applicant types, as strings.
+
+   ========================================================================== #}
+
+{%- import 'macros/time.html' as time -%}
+{% import 'macros/util/format/datetime.html' as dt -%}
+
+{%- macro city_str( city ) -%}
+{{ city.name ~ ', ' ~ city.state_id }}
+{%- endmacro -%}
+
+<section class="block
+                block__flush-top
+                block__sub
+                vacancy-announcement_details">
+    <h1>{{ value.title }}</h1>
+    <dl class="job-details">
```

Comment:
This `job-details` class appears to be unused and can be deleted. Unless there's some JS that ties in here somehow?

Reply:
Good catch, I think you're right that this is unused. I'll delete.